rpm/sailfish-browser.spec

```diff
@@ -22,6 +22,8 @@ BuildRequires:  qt5-qttools
 BuildRequires:  qt5-qttools-linguist
 BuildRequires:  gdb
 BuildRequires:  oneshot
+BuildRequires:  gtest-devel
+BuildRequires:  libgmock-devel
```

Comment:
IMO having ```BuildRequires``` at the top level is not good habit when the build requirement is not required by all packages that are going to be build. Thus, I'd specific this for test package only even thou "mock" and "test" are explaining this quite well :). More like a documentation thingie.

Ah right, ```Autoreq``` takes care of installation time dependency by checking compilation results.